supplementary/alica_ros1/alica_ros_turtlesim/etc/plans/Simulation.pml

```diff
@@ -1,15 +1,15 @@
 {
   "blackboard": [
     {
-      "access": "input",
+      "access": "protected",
```

Comment:
Could you please explain why we changes it? Just curious =)

Reply:
the plan designer (UI) has a check that 'input keys need to be mapped to some parent key' which is not satisfied here. Therefore changed it to "protected" which does not have this check